### DISCUSSION TITLE
Expand amount of domains that can be listed in the Domain Names field from 15 to something bigger

### DISCUSSION THREAD

**Discussion by Shocktrooper** [nginx-proxy/nginx-proxy#2323] [Ideas]
We currently have a lot of variations of the same domain that all point to a single domain
E.G. domain1.net, domain1,us domain1.com, domain1.org, domain1.ws, etc.  
All point to ForwardDomain.com

Currently it appears that the domain names field is limited to 15.  Unless there is a limitation with certificates, nginx itself or something else I would like to propose that we bump this to a higher number E.G. 100

This will allow us and others who have a lot of variations of domains all pointing at the same place to use a single records instead of splitting them out.

The following image is what you get if you try to enter more than 15
![image](https://github.com/nginx-proxy/nginx-proxy/assets/2646851/70de2dd4-3a00-4492-b1bd-38d75c848c24)

**Comment by SchoNie** [comment]
Your image seems to be from nginx-proxy-manager
Which is not this project.

So you should ask at https://github.com/NginxProxyManager/nginx-proxy-manager/ which is the location of nginx-proxy-manager.

**Comment by Shocktrooper** [comment]
gosh darned it.  My bad I didn't follow by bookmark
